public/js/test/utils/renderComponentFromFixture.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const React = require("react");
+const { DOM: dom, createElement } = React;
+const { Provider } = require("react-redux");
+const { fromJS, Map } = require("immutable");
+const { combineReducers } = require("redux");
+
+const fixtures = require("../fixtures");
+const configureStore = require("../../create-store");
+const reducers = require("../../reducers");
+
+function createStore(state = {}) {
+  return configureStore({})(combineReducers(reducers), state);
+}
+
+function getData(fixtureName) {
+  const fixture = fixtures[fixtureName];
+  return {
+    pause: Map({
+      pause: fromJS(fixture.pause.pause),
+      loadedObjects: fromJS(fixture.pause.loadedObjects),
+      frames: fixture.pause.frames
+    }),
+    sources: Map({
+      sources: fromJS(fixture.sources.sources)
+    })
```

Comment:
I see this data growing as we add more tests, but it should be consistent across tests as each field has a type